lib/ansible/modules/source_control/gitlab_deploy_key.py

```diff
@@ -9,35 +7,46 @@
 from __future__ import absolute_import, division, print_function
 __metaclass__ = type
 
-
 ANSIBLE_METADATA = {'metadata_version': '1.1',
                     'status': ['preview'],
                     'supported_by': 'community'}
 
-
 DOCUMENTATION = '''
----
 module: gitlab_deploy_key
 short_description: Manages GitLab project deploy keys.
 description:
      - Adds, updates and removes project deploy keys
 version_added: "2.6"
 options:
-  api_url:
+  server_url:
```

Comment:
This is going to break old users, think it's worth also accepting `api_url`? Or potentially using an alias and stripping off `/api`? That feels hackish, though. (also applies to `gitlab_hooks`)

Reply:
will be fixed in next commits